Library/Homebrew/cmd/reinstall.rb

```diff
@@ -57,6 +57,9 @@ def reinstall_args
           env:         :display_install_times,
           description: "Print install times for each formula at the end of the run.",
         }],
+        [:switch, "-g", "--git", {
```

Comment:
`-g` looks ambiguous, it's easy to confuse with `-g` for `--global` that `npm` has (and we don't). Also, `--git` doesn't look long tho

```suggestion
        [:switch, "--git", {
```

Reply:
I copied the `-g` from install: https://github.com/Homebrew/brew/blob/8a8de9e0303d78e04a6c48b552447831a3385d88/Library/Homebrew/cmd/install.rb#L105-L107

Would you prefer that `reinstall` only allows `--git`?  I'm fine either way, I was trying to maintain the illusion that the commands are mostly similar.

Reply:
Ah, ok, let's have it consistent then and leave it as it is. Thanks!